src/menu/menu.scss

```diff
@@ -94,6 +119,15 @@
       }
     }
   }
+
+  > .iui-divider {
```

Comment:
I would call this `iui-menu-divider` or something, and move it outside. It's specificity should be very low

Reply:
Renamed to `iui-menu-divider`.  Also introduced `iui-menu-item` so I can target the class directly rather than targeting the `li` element.  This allows me to remove the part that says `li:not(.iui-menu-divider)`.

Reply:
The specificity is still high. I'd move both of those things out into their own flat mixins, and declare in classes.scss

Reply:
> @FlyersPh9 Before any further review, can you verify the visuals? The spacing just looks off to me.

Since replacing the padding with a fixed height variable, the menu item height should match the height of the medium & large select component.